lib/streamlit/DeltaGenerator.py

```diff
@@ -2121,6 +2137,28 @@ def add_rows(self, data=None, **kwargs):
                 "Method requires exactly one dataset"
             )
 
+        # As we are using vega_lite for these deltas we have to reshape
```

Comment:
Replace `As we are using vega_lite for these deltas we have to reshape` with just `For some delta types we have to reshape`